src/App.css

```diff
@@ -20,4 +20,45 @@
 }
 table, thead, tbody {
 	display: block !important;
-}
\ No newline at end of file
+}
+
+.ol-control{
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.ol-zoom{
+  left: unset;
+  right: .5em;
+}
+
+.ol-rotate{
+  top: 4.875em;
+}
+
+.ol-geocoder{
+  position: unset !important;
+}
+
+.gcd-gl-btn{
+  height:2em !important;
+  width: 2em !important;
+}
+
+.gcd-gl-control{
+  position: relative;
+  float: left;
+  width: 16.5em !important;
+  background: none !important;
+}
+
+.gcd-gl-input{
+  width: 16em !important;
+}
```

Comment:
Don't bother finding other ways right now, but generally if you find yourself needing to put "!important"s in a lot of places, not a good sign. Can't say what the best way is with this open layers stuff, just something to keep in mind.